java-client/pom.xml

```diff
@@ -395,45 +390,6 @@
         <version>0.13</version>
       </plugin>
 
-      <plugin>
-        <groupId>com.diffplug.spotless</groupId>
-        <artifactId>spotless-maven-plugin</artifactId>
-        <version>2.12.2</version>
-        <configuration>
-          <formats>
-            <format>
-              <includes>
-                <include>*.md</include>
-                <include>.gitignore</include>
-              </includes>
-              <trimTrailingWhitespace/>
-              <endWithNewline/>
-              <indent>
-                <tabs>true</tabs>
-                <spacesPerTab>4</spacesPerTab>
-              </indent>
-            </format>
-          </formats>
-          <java>
-            <excludes>
-              <exclude>src/main/java/org/apache/infra/pegasus/apps/*</exclude>
-              <exclude>src/main/java/org/apache/infra/pegasus/replication/*</exclude>
-            </excludes>
-            <googleJavaFormat>
-              <version>1.7</version>
-              <style>GOOGLE</style>
-            </googleJavaFormat>
-          </java>
-        </configuration>
-        <executions>
-          <execution>
-            <goals>
-              <goal>check</goal>
-            </goals>
-            <phase>compile</phase>
-          </execution>
-        </executions>
-      </plugin>
```

Comment:
if you delete the plugin, how to format code to match ci for user?

Reply:
In the role of a basic component, Pegasus java-client should better depend on less packages. I think check and format code style is not necessary to it.

Reply:
1.  code style check these don't feel good to write in our logic code
2.  we can configure code format plugin in ieda

Reply:
> In the role of a basic component, Pegasus java-client should better depend on less packages. I think check and format code style is not necessary to it.

I see, but I want to konw how format it to accord with the ci after we code completed?  

> 2\. we can configure code format plugin in idea

Maybe you should make sure it can work well and add the solution in readme.

Reply:
ci will only check the style of the newly submitted code, not check the unmodified code；
and use checkstyle v0.6.1 to check it later，when it can work well and I will write in redme to remind everyone to use checkstyle

Reply:
I write new code, and checkstyle not pass, tell me how to do can let the check pass?